aten/src/ATen/native/transformers/transformer.cpp

```diff
@@ -106,7 +106,7 @@ Tensor transformer_encoder_layer_forward(
      x = at::linear(x, qkv_weight, qkv_bias);
      auto x_size_0 = x.size(0);
      x = x.view({x_size_0, -1, 3, num_heads, embed_dim / num_heads});
-     x = flash_attention_helper(x, x, x, 0.0, false, false);
+     x = std::get<0>(flash_attention_helper_packed(x, x, x, 0.0, false, false));
```

Comment:
This is the only call site, why update the signature to return a tuple?

Reply:
yeah this was copied in from another PR I will undo